### DISCUSSION TITLE
Unlock a user account

### DISCUSSION THREAD

**Discussion by Bart-De-Pauw** [dbeaver/dbeaver#12843] [DBeaver Development]
Dear,

As an Oracle DBA, I recently had to unlock a user account, but I could not find any possibility to do this using the GUI.
I had to use plain SQL to achieve this.

What are the possibilities in DBeaver (I'm using dbeaver21.0.4 ce).
What are the future plans in terms of more advanced (Oracle) DBA features that will be provided via the GUI or is DBeaver more considered to be a Developer tool?

Best regards,
Bart

**Comment by LonwoLonwo** [comment]
Hello @Bart-De-Pauw 
If you create a ticket as a feature request here https://github.com/dbeaver/dbeaver/issues, then he gets into our workflow.
Please add more details and examples there, and when we can quickly handle the task.
